server/src/cg/group4/util/IpResolver.java

```diff
@@ -14,15 +15,18 @@
     /**
      * String to be used for the URL to check external IP.
      */
-    protected final String cVerifyURl = "http://checkip.amazonaws.com";
+    private static String cVerify = "http://checkip.amazonaws.com";
```

Comment:
Something with checkstyle or findbugs complaining. Changed it back to protected and did not see the change again, so no clue.